code/Extension/FormSpamProtectionExtension.php

```diff
@@ -106,7 +106,7 @@ public function enableSpamProtection($options = array())
         // set custom mapping on this form
         $protector = self::get_protector($options);
```

Comment:
Can you clarify what's the scenario where `enableSpamProtection()` would be called and `get_protector` would return `null`?

Maybe throwing an exception is the better outcome here.

If there's a legitimate use case for this situation to occur, we should probably have a unit test to cover it.